pom.xml

```diff
@@ -59,7 +59,7 @@
         <revision>2.29</revision>
         <gitHubRepo>jenkinsci/pipeline-stage-view-plugin</gitHubRepo>
         <bom>2.289.x</bom>
```

Comment:
I think you forgot to update the BOM to match.

Reply:
oh my bad! Good catch! Thanks
fixed with https://github.com/jenkinsci/pipeline-stage-view-plugin/pull/263